src/init.cpp

```diff
@@ -1861,16 +1855,7 @@ bool AppInitMain(const CoreContext& context, NodeContext& node, interfaces::Bloc
         {
             // Create and register activeMasternodeManager, will init later in ThreadImport
             ::activeMasternodeManager = std::make_unique<CActiveMasternodeManager>(*node.connman);
```

Comment:
btw, why it is here, not in llmq-context? without llmq context `activeMasternodeManager` has no usage; and it is heavily used in llmq-context.

(not a blocker to get merged, just an opener for a discussion, either way is fine for me)

Reply:
Because `activeMasternodeManager`

* Relies on `CConnman` and `CDeterministicMNManager` to exist ([source](https://github.com/dashpay/dash/blob/a7e61114f8473cbf7e2fd3e8ded6b2de5eb3ff9f/src/masternode/node.h#L56-L59))
* It doesn't depend on LLMQ managers to exist (unlike _most_ LLMQ managers, which depend on each other)
* It's used in `MNAuth` ([source](https://github.com/dashpay/dash/blob/a7e61114f8473cbf7e2fd3e8ded6b2de5eb3ff9f/src/evo/mnauth.cpp#L22-L59)) and `CGovernanceManager` ([source](https://github.com/dashpay/dash/blob/a7e61114f8473cbf7e2fd3e8ded6b2de5eb3ff9f/src/governance/governance.cpp#L763-L776))